.github/workflows/ci.yml

```diff
@@ -57,35 +57,21 @@ jobs:
             -   name: Checkout source
                 uses: actions/checkout@v3
 
-            -   name: Get symfony flex version
-                uses: actions/github-script@v3.1
-                id: flex-version
-                with:
-                    script: 'return "${{ matrix.php }}" == "7.0" ? "~1.6.0" : "^1.11"'
-                    result-encoding: string
-
             -   name: PHP setup
                 uses: shivammathur/setup-php@v2
                 with:
                     php-version: ${{ matrix.php }}
                     ini-values: zend.assertions=1, error_reporting=-1, display_errors=On, log_errors_max_len=0
                     coverage: ${{ matrix.coverage || 'none' }}
 
-            -   name: Downgrade composer for old Symfony Flex
-                if: steps.flex-version.outputs.result == '~1.6.0'
-                run: composer self-update --1
-
             -   name: Configure composer stability
                 if: matrix.composer-stability
                 run: composer config minimum-stability ${{ matrix.composer-stability }}
 
-            -   name: "Disallow Symfony Flex to run as a Composer plugin (global)"
```

Comment:
looks like we handled it in parallel
https://github.com/php-coveralls/php-coveralls/pull/393#issue-3587998113